tests/anoph/conftest.py

```diff
@@ -43,6 +45,272 @@ def simulate_genome(*, path, contigs, low, high):
         seq = simulate_contig(low=low, high=high)
         root.create_dataset(name=contig, data=seq)
     zarr.consolidate_metadata(path)
+    return root
+
+
+class Gff3Simulator:
```

Comment:
This is a bit more complicated than I would've liked but tried to take the simplest route to generating some semi-realistic genome features data.